### PR TITLE
[libvpx] Add pkgconfig

### LINKED ISSUE #19346: [libvpx] Missing vcpkg_fixup_pkgconfing
libvpx install invalid pc files.

Found due to:
```
CMake Error in /mnt/vcpkg-ci/buildtrees/qtwebengine/x64-linux-dbg/CMakeFiles/CMakeTmp/CMakeLists.txt:
  Imported target "PkgConfig::VPX" includes non-existent path

    "/mnt/vcpkg-ci/packages/libvpx_x64-linux/debug/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.
```

### PR DESCRIPTION
**Pull request by NancyLi1013**
**Describe the pull request**

- #### What does your PR fix?  
  Fixes #19346

Add `vcpkg_fixup_pkgconfig()` to fix the paths in .pc files.

Note: No feature needs to test.

### CODE REVIEW

File: ports/libvpx/portfile.cmake [hunk 1]
```diff
@@ -238,9 +238,12 @@ else()
             LOGNAME install-${TARGET_TRIPLET}-dbg
         )
 
-        file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
-        file(REMOVE ${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a)
+        file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+        file(REMOVE "${CURRENT_PACKAGES_DIR}/debug/lib/libvpx_g.a")
     endif()
+    
+    vcpkg_fixup_pkgconfig()
```
Comment (Neumann-A) [inline ports/libvpx/portfile.cmake:245]: No pc files in the windows path? Probably should generate the pc files there also?

Comment (NancyLi1013) [reply ports/libvpx/portfile.cmake:245]: Yes, I checked there were no .pc files generated. So I add this only for Unix platform.

Comment (Neumann-A) [reply ports/libvpx/portfile.cmake:245]: In generally ports should generate pc files on all platforms. So vcpkg needs to patch the behavior of this port to do so on windows. (Required e.g. for meson usage. )

Comment (NancyLi1013) [reply ports/libvpx/portfile.cmake]: Thanks for your reminder. I have added .pc file on Windows now.

**Comment by NancyLi1013** [comment]
Since there is no .pc file on Windows. And vpx.pc on Unix is generated from libs.mk. The content in vpx.pc is written like this:

https://github.com/webmproject/libvpx/blob/master/libs.mk#L377-L399

`libvpx ` has different ways to build on Windows and on Unix. So I add `vpx.pc.in` file on Windows.

File: ports/libvpx/vpx.pc.in [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+exec_prefix=${prefix}
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: vpx
+Description: WebM Project VPx codec implementation
+Version: @LIBVPX_VERSION@
+Requires:
+Conflicts:
+Libs: -L"${libdir}" -lvpx -lm
```
Comment (dg0yt) [inline ports/libvpx/vpx.pc.in:12]: IIUC you are add this for Windows. Is there `-lm` on Windows?

And for non-Windows, is there already a pc file?

Comment (NancyLi1013) [reply ports/libvpx/vpx.pc.in:12]: Yes, need to remove it on Windows.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**